.gitignore

```diff
@@ -41,3 +41,4 @@ Makefile.dep
 compile_commands.json
 redis.code-workspace
 .cache
+.cscope.*
```

Comment:
interested to see your dev setup :D

Reply:
Vim + ctags + cscope :)

Reply:
Just generally, we should avoid such unrelated changes ideally.